pycytominer/operations/correlation_threshold.py

```diff
@@ -48,7 +48,7 @@ def correlation_threshold(
 
     # Subset dataframe and calculate correlation matrix across subset features
     if samples != "all":
-        population_df.query(samples, inplace=True)
+        population_df = population_df.query(samples)
```

Comment:
Consider also adding or enhancing the code comment above this if-branch to provide a description of what this is doing.